docs/summary.md

```diff
@@ -0,0 +1,7 @@
+# Summary
+
+It follows a list of all the implemented workflows:
+
+- [`publish-next-to-ghpages`](./publish-next-to-ghpages.md)
+
```

Comment:
Remove blank lines, otherwise they will be two separate lists of a single item each